<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KMapWindow</class>
 <widget class="QMainWindow" name="KMapWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuGenerate">
    <property name="title">
     <string>Generate</string>
    </property>
    <addaction name="actionSOP"/>
    <addaction name="actionPOS"/>
   </widget>
   <addaction name="menuGenerate"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionSOP">
   <property name="text">
    <string>SOP</string>
   </property>
  </action>
  <action name="actionPOS">
   <property name="text">
    <string>POS</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
